tracing-subscriber/README.md

```diff
@@ -17,7 +17,7 @@ Utilities for implementing and composing [`tracing`][tracing] subscribers.
 [crates-badge]: https://img.shields.io/crates/v/tracing-subscriber.svg
 [crates-url]: https://crates.io/crates/tracing-subscriber
 [docs-badge]: https://docs.rs/tracing-subscriber/badge.svg
-[docs-url]: https://docs.rs/tracing-subscriber/0.2.3
+[docs-url]: https://docs.rs/tracing-subscriber/0.2.4
```

Comment:
One of these days we should this version thing/dance—docs.rs can how load the latest release. I also think  crates.io links to the corresponding version on docs.rs.

Reply:
You can use `/crate/latest/` on docs.rs. I try to _avoid_ doing this so that each version's README on crates.io links to the corresponding version.